src/metabase/driver/postgres.clj

```diff
@@ -163,7 +163,13 @@
     driver.common/advanced-options-start
     (assoc driver.common/additional-options
            :placeholder "prepareThreshold=0")
-    driver.common/default-advanced-options]
+    driver.common/default-advanced-options
+    {:name         "database-enable-actions"
+     :type         :boolean
+     :default      false
+     :display-name (deferred-tru "Enable experimental actions")
```

Comment:
@camsaul I used `deferred-tru` because that's what `common.clj` uses for settings but the rest of the code here uses `trs`. Is one preferred over the other? Also, is this setting name supposed to mirror the `defsetting` in #22559 ?

@camsaul @kulyk is this the language we want to use on the database settings page?

Reply:
> is this the language we want to use on the database settings page?

I think this one works at the moment 👍